test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -184,6 +184,78 @@ public void TestGreyscaleTransformImages()
             Done();
         }
 
+        [Fact]
+        public void TestGrayScaleInMemory()
+        {
+            var imagesDataFile = SamplesUtils.DatasetUtils.DownloadImages();
+
+            var data = ML.Data.CreateTextLoader(new TextLoader.Options()
+            {
+                Columns = new[]
+                {
+                    new TextLoader.Column("ImagePath", DataKind.String, 0),
+                    new TextLoader.Column("Name", DataKind.String, 1),
+             }
+            }).Load(imagesDataFile);
```

Comment:
Simplify the test by creating bitmaps on the fly - for example 100% red or blue bitmap.  This would make it more realistic and simple code.